packages/synapse-react-client/stories/ErrorBanner.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { ErrorBanner } from '../src/components/error/ErrorBanner'
+import { SynapseClientError } from '../src/utils/SynapseClientError'
+
+const meta = {
+  title: 'UI/ErrorBanner',
+  component: ErrorBanner,
+} satisfies Meta
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const SynapseClientErrorDemo: Story = {
+  render: args => (
+    <>
+      <div>Note: login to see the unauthorized SynapseClientError.</div>
+      <ErrorBanner {...args} />
+    </>
+  ),
```

Comment:
If you don't need to pass anything different to the story, you can use a decorator instead of the render option.

```suggestion
  decorators: [Story => (
    <>
      <div>Note: login to see the unauthorized SynapseClientError.</div>
      <Story />
    </>
  )],
```

Reply:
Oh very cool! This is great